financial-templates-lib/clients/TokenBalanceClient.js

```diff
@@ -28,23 +28,23 @@ class TokenBalanceClient {
   }
 
   // Delete all data within the client.
-  clearState = async () => {
+  clearState = () => {
     this.tokenBalances = { collateralBalances: {}, syntheticBalances: {}, etherBalances: {} };
     this.accountMonitorList = [];
   };
 
-  getCollateralBalance = address => {
```

Comment:
Previously calling these functions would return a `null` value if the client had not yet been updated. This meant that client implementers had to deal with the null case and wait until the following update round to get a balance. In this update, the balances are retrieved immediately after calling these functions to accommodate the single fire serverless execution.

Reply:
nit: same format as above

Reply:
This function should also be in traditional non-arrow style